src/domain/mocks/handlers.ts

```diff
@@ -303,6 +304,89 @@ export const handlers = [
     return res(ctx.delay(), ctx.status(204));
   }),
 
+  rest.get(`${apiUrl}/kayttajat/:id/deleteInfo`, async (req, res, ctx) => {
+    const { id } = req.params;
+    if (id === '3fa85f64-5717-4562-b3fc-2c963f66afa7') {
+      return res(
+        ctx.delay(),
+        ctx.status(200),
+        ctx.json<DeleteInfo>({
+          activeHakemukset: [
+            { nimi: 'Hakemus 1', alluStatus: 'PENDING', applicationIdentifier: 'JS2300001' },
+            { nimi: 'Hakemus 2', alluStatus: 'HANDLING', applicationIdentifier: 'JS2300002' },
+          ],
+          draftHakemukset: [],
+          onlyOmistajanYhteyshenkilo: false,
+        }),
+      );
+    }
+    if (id === '3fa85f64-5717-4562-b3fc-2c963f66afa8') {
+      return res(
+        ctx.delay(),
+        ctx.status(200),
+        ctx.json<DeleteInfo>({
+          activeHakemukset: [
+            { nimi: 'Hakemus 1', alluStatus: 'PENDING', applicationIdentifier: 'JS2300001' },
+            { nimi: 'Hakemus 3', alluStatus: 'PENDING', applicationIdentifier: 'JS2300003' },
+          ],
+          draftHakemukset: [],
+          onlyOmistajanYhteyshenkilo: false,
+        }),
+      );
+    }
+    if (id === '3fa85f64-5717-4562-b3fc-2c963f66afa9') {
+      return res(
+        ctx.delay(),
+        ctx.status(200),
+        ctx.json<DeleteInfo>({
+          activeHakemukset: [],
+          draftHakemukset: [
+            { nimi: 'Hakemus 4', alluStatus: null, applicationIdentifier: 'JS2300004' },
+            { nimi: 'Hakemus 5', alluStatus: null, applicationIdentifier: 'JS2300005' },
+          ],
+          onlyOmistajanYhteyshenkilo: false,
+        }),
+      );
+    }
+    if (id === '3fa85f64-5717-4562-b3fc-2c963f66afb1') {
```

Comment:
This user, Tauno Testinen, is not OMISTAJA but TOTEUTTAJA. Maybe his role should be changed in `users-data.json`? I am not sure about the other users and their roles but this one I noticed.

Reply:
Good point. Made changes to `users-data.json`.